pkgs/by-name/vi/vintagestory/package.nix

```
@@ -21,11 +21,11 @@
 
 stdenv.mkDerivation rec {
   pname = "vintagestory";
-  version = "1.19.8";
+  version = "1.20.0";
```

```suggestion
  version = "1.20.1";
```
Came out six hours ago with bug fixes. If we're bumping, might as well bump to the latest version.